src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubSCMSource/help-repositoryUrl_zh_CN.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    需要扫描的代码仓库。
```

Comment:
Where did this come from? Unless you are sure it makes sense in the new context, I wouldn't add any localizations.